baybe/acquisition/base.py

```diff
@@ -39,15 +39,19 @@ def to_botorch(
         self,
         surrogate: Surrogate,
         searchspace: SearchSpace,
-        train_x: pd.DataFrame,
-        train_y: pd.DataFrame,
+        measurements: pd.DataFrame,
     ):
         """Create the botorch-ready representation of the function."""
         import botorch.acquisition as botorch_acqf_module
 
         acqf_cls = getattr(botorch_acqf_module, self.__class__.__name__)
         params_dict = filter_attributes(object=self, callable_=acqf_cls.__init__)
 
+        train_x, train_y = (
+            surrogate.transform_inputs(measurements),
+            surrogate.transform_targets(measurements),
+        )
+
```

Comment:
are you sure the `best_f` bug has been fixed? the best_f line below is still the same, it gets the unscaled target

Reply:
Alright, two things here:
1. I thought again about it and now came to the (hopefully right?) conclusion that it never was a bug in the first place. While the surrogates (including non-GP ones) did some scaling to `Y`, that was all hidden in the decorated `posterior` method. That is, what the acquisition function got to see was only the output of `posterior`, which is already back on the correct scale, and those values would be compared to `best_f`, which is also on the original scale. Because of the way how currently stuff is intertwined (i.e., recommenders operating on both reps, scaling partially done here and there, ...), things are really hard to grasp though. I expect that this will become much clearer after this series of refactoring PRs, and then we should have one final check. If you agree with all that, I'll remove the changelog entry, and please also try to remind me to double-check later (will additionally put a TODO comment next to `best_f`).
2. I also noticed that the PR was far from finished. Have added two additional commits and would kindly ask you to review them

Reply:
ok when looking at it i also had the hunch that it might not have been a bug in the first place, so resolve?

Reply:
Yes, will resolve and we give it another critical investigation at the end of this dev branch